code/__HELPERS/logging.dm

```
@@ -81,8 +81,10 @@
  * Helper proc to log attacks or similar events between two mobs.
  */
 /proc/add_attacklogs(var/mob/user, var/mob/target, var/what_done, var/object = null, var/addition = null, var/admin_warn = TRUE)
-	var/user_txt = (user ? "[user][user.ckey ? " ([user.ckey])" : ""]" : "\<NULL USER\>")
-	var/target_txt = (target ? ismob(target) ? "[target][target.ckey ? " ([target.ckey])" : ""]" : "[target]" : "\<NULL TARGET\>")
+	if (!user) // That should never happen
```

If it should never happen then why add this check? It just hides the error